views/styles.html

```diff
@@ -340,5 +340,16 @@ <h2 class="_style-guide--title">Tabular Data</h2>
     {{table(tableData)}}
   </section>
 
+  <section class="_style-guide--group">
+    <h2 class="_style-guide--title">All Content Types</h2>
+    {% from "components/card.html" import card %}
+    <div class="card--group">
```

Comment:
Layout needs to match the file name (and therefore the layout name) in the styling